jplag.endtoend-testing/src/main/java/de/jplag/endtoend/helper/JsonHelper.java

```diff
@@ -12,6 +12,11 @@
 import de.jplag.endtoend.constants.TestDirectoryConstants;
 import de.jplag.endtoend.model.ResultDescription;
 
+/**
+ * Helper class for serialization and deserialization of the used json format into the correct record classes. The
+ * serialization/deserialization is enabled using the Jackson library. The record classes used can be found at
+ * de.jplag.endtoend.model.
```

Comment:
```suggestion
 * {@link de.jplag.endtoend.model}.
```

I think this should work